R/pkg/inst/tests/testthat/test_Serde.R

```diff
@@ -28,6 +28,10 @@ test_that("SerDe of primitive types", {
   expect_equal(x, 1)
   expect_equal(class(x), "numeric")
 
+  x <- callJStatic("SparkRHandler", "echo", 1380742793415240)
```

Comment:
I did some google search. R can't specify `bigint` type. So, we can't directly test `bigint` type.

We can remove the tests above, as we added `schema` tests and scala API tests.